html/php-pre-challenge3/index.php

```diff
@@ -1,6 +1,78 @@
 <?php
-$limit = $_GET['target'];
+try{
+    //データベース接続
+    $dsn = 'mysql:dbname=test;host=mysql';
+    $dbuser = 'test';
+    $dbpassword = 'test';
+    $dbh=new PDO($dsn,$dbuser,$dbpassword);
+    //SQL実行小さい順
+    $sql = 'SELECT value FROM prechallenge3 ORDER BY value  ASC';
+    $stmt=$dbh->query($sql);
+    $result=$stmt->fetchall(PDO::FETCH_ASSOC);
+    //結果を配列に変換
+    $dbArray = array_column($result,'value');    
```

Comment:
```suggestion
    $dbArray = array_column($result, 'value');
    foreach ($dbArray as &$value) {
        $value = intval($value);
    }
```

DBから取り出した値を文字列のまま計算してしまっているので整数型にキャストしましょう。
あと行の末尾にスペースがあるのはよくないですね。これも自動整形で消せます。

Reply:
22bc1ac0f7b5e660a556f469fbf9263e3c39b7ea
このコミットで対応。
整数型にキャストしました！